book/src/prefabs/how_to_define_prefabs_aggregate.md

```diff
@@ -35,95 +30,98 @@ If you intend to include a [`Component`] that has not yet got a corresponding [`
 
     In these examples, `Named`, `Position`, and `Weapon` all derive [`PrefabData`].
 
-    * *Superset* prefab data:
-
-        ```rust,edition2018,no_run,noplaypen
-        # extern crate amethyst;
-        # extern crate derivative;
-        # extern crate serde;
-        # extern crate specs_derive;
-        #
-        # use amethyst::{
-        #     assets::{PrefabData, ProgressCounter},
-        #     core::Named,
-        #     derive::PrefabData,
-        #     ecs::{
-        #         storage::{DenseVecStorage, VecStorage},
-        #         Component, Entity, WriteStorage,
-        #     },
-        #     prelude::*,
-        #     Error,
-        # };
-        # use derivative::Derivative;
-        # use serde::{Deserialize, Serialize};
-        # use specs_derive::Component;
-        #
-        # #[derive(Clone, Copy, Component, Debug, Default, Deserialize, Serialize, PrefabData)]
-        # #[prefab(Component)]
-        # #[serde(deny_unknown_fields)]
-        # pub struct Position(pub f32, pub f32, pub f32);
-        #
-        # #[derive(Clone, Copy, Component, Debug, Derivative, Deserialize, Serialize, PrefabData)]
-        # #[derivative(Default)]
-        # #[prefab(Component)]
-        # #[storage(VecStorage)]
-        # pub enum Weapon {
-        #     #[derivative(Default)]
-        #     Axe,
-        #     Sword,
-        # }
-        #
-        /// All fields implement `PrefabData`, and are wrapped in `Option<_>`.
-        ///
-        /// **Note:** If a field is not specified in the prefab, it will default
-        /// to `None`.
-        #[derive(Debug, Default, Deserialize, Serialize, PrefabData)]
-        #[serde(default)]
-        #[serde(deny_unknown_fields)]
-        pub struct CustomPrefabData {
-            player: Option<Named>,
-            weapon: Option<Weapon>,
+    ```rust,edition2018,no_run,noplaypen
+    # extern crate amethyst;
+    # extern crate derivative;
+    # extern crate serde;
+    # extern crate specs_derive;
+    #
+    # use amethyst::{
```

Comment:
Wouldn't it make sense to show these imports (and maybe even the two structs below)?

Reply:
Yeah, I was following the style of the existing writeup but I think you're right, showing more is useful. I'll open it up.

Reply:
I included the structs but now that I look at this again I see that the imports are already visible in a block just above. I think that's cleaner that showing them twice so I'm going to leave it that way.

Reply:
Ah, you're right, sorry. I've missed that.